test/TestServerProjectsLowLevel/dpg-customization/Models/Input.Serialization.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
```

Comment:
If we are going to use this as an example for service team's to follow, we should indicate that these files are not generated.

Reply:
(To be clear: we still need to decide whether this is a proof-of-concept, or an example for service teams to follow)

Reply:
Do we still need to remove this as we are now implementing full grow-up story.

Reply:
Models are not auto-generated in the grow-up story.  We'll do work in Copper to support generation of models.  The full grow-up story for Nickel is that models are hand-authored and must be approved by the arch board in order to GA.

Reply:
We will generate models for scenario 4 as part of the work we do in Copper.  This still needs to be designed and a variety of other issues need to be addressed as part of finalizing the generation story.  Until we close those issues, we will "hand-write" models for scenario 4.

That doesn't mean you can't use the generator to create the models.  I just don't want us to give the impression to partner teams that they can generate and ship models as part of the grow-up story, until we are able to say that officially.

Reply:
Clear now. It makes sense!